docs/manage-your-hypervisor/index.rst

```diff
@@ -17,9 +17,20 @@ your request.
 
 This chapter includes the following sections:
 
+- :ref:`Manage vSphere clusters <manage-vsphere-clusters>`
+- :ref:`Upgrade ESXi version <upgrade-esxi-version>`
+- :ref:`Manage hypervisor hardware <manage-hypervisor-hardware>`
+- :ref:`Manage hypervisor networking <manage-hypervisor-networking>`
+- :ref:`Manage hypervisor storage <manage-hypervisor-storage>`
+- :ref:`Plan hypervisor capacity <plan-hypervisor-capacity>`
+
+
+
 .. toctree::
    :maxdepth: 2
+   :hidden:
 
+   self
```

Comment:
"self" only needed to be included in the base index.rst file, not every one

Reply:
By base, I mean the index.rst file that's in the docs folder, so none of the index files in the sub folders need to have that in there